Update WG Lifecycle

lehors opened the pull request:
Introduce Sandbox stage, and rename other stages to: Incubating, Graduated, and Archived to align with other lifecycles.

steiza commented on process/working-group-lifecycle.md:
```diff
@@ -8,14 +8,29 @@ However, if no existing effort covers the topic feel free to make use of the exi
 
-The WG Life Cycle begins with establishing commitments from other interested contributors then making a proposal to the TAC. If accepted, the WG will begin as `Incubating`. This is a boot strapping phase for the group to adopt OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be fully recognized in OpenSSF. Either upon completing its objectives or no lon ...
+The WG Life Cycle begins with establishing commitments from other interested contributors then making a proposal to the TAC. If accepted, the WG will begin as `Samdbox`. This is a boot strapping phase for the group to define and document what their goals are. 
+adopt OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be fully recognized in OpenSSF. Either upon completing its objectives or no longer making forward progress a project will move to `End-of-Life`. The details for each phase follow.
```
It looks like there's a dangling sentence fragment here "adopt OpenSSF community policies and norms."

steiza replied:
Also let's swap `Active` for `Graduated` and `End-of-Life` for `Archived`?

lehors replied:
Done.

steiza commented on process/working-group-lifecycle.md:
```diff
@@ -8,14 +8,29 @@ However, if no existing effort covers the topic feel free to make use of the exi
+## Once `Sandbox`:
+
+* The TAC will add the WG to the list of WGs to its README.
+* If the WG has meetings:
```
I was confused by this at first, because I was thinking "of course a WG must have meetings!" But I think what we're saying is "If a WG has meetings **at the sandbox stage**"? If so, we might want to make that more explicit.

lehors replied:
I added your piece to make it clear.

steiza commented on process/working-group-lifecycle.md:
```diff
@@ -8,14 +8,29 @@ However, if no existing effort covers the topic feel free to make use of the exi
+ 
+## To become `Incubating`:
+
+* The WG should have a charter or mission statement for review by TAC
```
I would argue this is a "must" not a "should" : )

lehors replied:
Agreed.

steiza reviewed (commented):
Thank you for taking this on! I have a bunch of minor comments, but otherwise this is great.

lehors commented:
Sorry, somehow I had committed an intermediary version and not my latest changes. I've now done that and addressed your comments at the same time. Thanks for the quick review!

steiza reviewed (approved):
Looks good to me!

mlieberman85 reviewed (approved):
LGTM

SecurityCRob reviewed (approved):
+1

SecurityCRob commented:
Merging - 50% voted in favour so far

SecurityCRob commented:
me-- forgot to merge first.
